multiqc/qualimap/__init__.py

```diff
@@ -40,12 +48,10 @@ def __init__(self):
                 s_name = self.clean_s_name(s_name, root)
                 if s_name in qualimap_raw_data:
                     log.debug("Duplicate sample name found! Overwriting: {}".format(s_name))
-                
+
                 qualimap_raw_data[s_name] = {}
-                qualimap_raw_data[s_name]['reports'] = {os.path.splitext(r)[0]: os.path.join(root, 'raw_data_qualimapReport', r) \
-                    for r in os.listdir(os.path.join(root, 'raw_data_qualimapReport'))}
-                qualimap_raw_data[s_name]['plots'] = {os.path.splitext(r)[0]: os.path.join(root, 'images_qualimapReport', r) \
-                    for r in os.listdir(os.path.join(root, 'images_qualimapReport'))}
+                qualimap_raw_data[s_name]['reports'] = {os.path.splitext(r)[0]: os.path.join(root, raw_data_dir, r) \
```

Comment:
In fact, if you're looking for files with specific filenames (keys looked for below), can't you just look for files with those filenames?